g_docs.go

```diff
@@ -654,6 +628,33 @@ func parserComments(comments *ast.CommentGroup, funcName, controllerName, pkgpat
 	return nil
 }
 
+func consumes(a string) []string {
+	switch a {
+	case "json":
+		return []string{ajson}
+	case "xml":
+		return []string{axml}
+	case "plain":
+		return []string{aplain}
+	case "html":
+		return []string{ahtml}
+	case "thrift_binary":
+		return []string{content_type_thrift_binary}
+	case "thrift_json":
+		return []string{content_type_thrift_json}
+	case "thrift_webcontent_binary":
+		return []string{content_type_thrift_binary_webcontent_v1}
+	case "thrift_webcontent_json":
+		return []string{content_type_thrift_json_webcontent_v1}
+	case "multipart":
+		return []string{contentTypeMultipartFormData}
+	case "form":
+		return []string{contentTypeFormURLEncoded}
+	}
```

Comment:
I believe this breaks the "backwards" compatibility: before `form` was producing `multipart/form-data` now it produces `application/x-www-form-urlencoded`. I don't know if this is a bug and was fixed in the meanwhile in original bee, because here https://github.com/beego/beedoc/blob/master/en-US/advantage/docs.md#application-comment under `@Parameter` it states the opposite :(

> formData means the parameter sends by POST ( set Content-Type to application/x-www-form-urlencoded ) .

I would rather keep our old buggy implementation. Assuming that we will switch to Chi http handler soon'ish and the source of truth will become these annotations powered by swaggo (and assuming swaggo does it right).

Reply:
Makes sense! Addressed in https://github.com/zalora/bee/pull/37/commits/76d4e865858d4c184d84ce7f7a62e59fcbc9eb1d